src/lib/CRSF/CRSF.cpp

```diff
@@ -813,11 +813,12 @@ bool CRSF::UARTwdt()
 void ICACHE_RAM_ATTR CRSF::ESP32uartTask(void *pvParameters)
 {
     DBGLN("ESP32 CRSF UART LISTEN TASK STARTED");
-    CRSF::duplex_set_TX();
+    noInterrupts();
```

Comment:
I've run at least a dozen start/stop WiFi from LUA which is how I normally reproduce this issue.
Seems that this line should be `portDISABLE_INTERRUPTS();`
And after I make these changes, so far I have not reproduced the crash on my setup.

Reply:
thanks for testiing, fixed!